hw2/05-form.html

```diff
@@ -1,20 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>05 - Form</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="05-form.js"></script>
-  </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
-    </main>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>05 - Form</title>
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+		/>
+		<script defer src="05-form.js"></script>
+	</head>
+	<body class="d-flex justify-content-center align-content-center mt-5 bg-dark">
+		<main class="bg-light w-25 rounded">
+			<form class="form" id="form" action="/submit">
+				<div style="text-align: center">
+					<h1 class="title mt-4" id="title">Exercise 05 - Form</h1>
+				</div>
+
+				<div class="name" id="nameip">
```

Comment:
I think you can do this with much fewer div elements. You should try using a layout to help.

Reply:
I will try it for future assignments for sure.